topics/statistics/tutorials/galaxy-ludwig/tutorial.md

```diff
@@ -0,0 +1,467 @@
+
+The rationale on how this file was constructed for this dataset is the following:
+i) The model takes images as input and uses a stacked convolutional neural network (CNN) to extract features.
+ii) It consists of two convolutional layers followed by a fully connected layer, with dropout applied to both the second convolutional layer and the fully connected layer to reduce overfitting. 
+iii) The split configuration is used to divide the dataset based on the 'split' column rather than using a random split (which is the default). In this configuration, rows labeled with '0' are included in the training set, while rows labeled with '2' are included in the testing set. If a validation category is needed, rows are labeled with '1'. 
+iv) The model is trained to classify images into categories (e.g., recognizing digits), and it will train for 5 epochs.
+
+![Config.yaml file snapshot](../../images/galaxy-ludwig/explain_config.png "Config.yaml file snapshot.")
+
+# GALAXY-LUDWIG TOOL 
+
+Ludwig simplifies the complexities of machine learning by automating essential steps such as data preprocessing, model architecture selection, hyperparameter tuning, and device management. This streamlined approach is particularly beneficial for Galaxy users who are more interested in addressing their scientific questions than in navigating the intricacies of machine learning workflows.
+
+## Prepare environment and get the data 
+
+> <hands-on-title> Environment and Data Upload </hands-on-title>
+>
+> 1. Create a new history for this tutorial. If you are not inspired, you can name it *image classifier*.
+>
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+>
+> 2. Import the files from Zenodo or from the shared data library
+>
+>    ```
+>    https://zenodo.org/records/13292895/files/mnist_images.zip
+>    https://zenodo.org/records/13357260/files/mnist_dataset.csv 
+>    https://zenodo.org/records/13713095/files/config.yaml
+>    ```
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+> 3. Check that the data format assigned for each file is **zip**, **csv**, and **yaml**, respectively.
+>    If it is not, follow the Changing the datatype tip.
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="datatypes" %}
+>
+> 5. Add a tag (`Stacked CNN`) to the dataset corresponding to `config.yaml`
+>    This is important to trace back on what configurations the model was built on.
+>
+>    {% snippet faqs/galaxy/datasets_add_tag.md %}
+>
+{: .hands_on}
+
+## Ludwig Experiment
+
+> <hands-on-title> Task description </hands-on-title>
+>
+> 1. {% tool [Ludwig Experiment](ludwig_experiment) %} with the following parameters:
+>    - {% icon param-file %} *"Select the dataset containing model configuration"*: `config.yaml`
+>    - {% icon param-file %} *"Input dataset"*: `mnist_dataset.csv`
+>    - {% icon param-file %} *"Raw data"*: `mnist_images.zip`
+>
+>    > <comment-title> short description </comment-title>
```

Comment:
you probably want to update or remove this comment title

Reply:
furthermore, I think we can assume people have enough familiarity with Galaxy to know to push the "RUN TOOL" button, so the box can also just be removed completely

Reply:
I agree. I removed it all.